server/liveblog/embed/embed.py

```diff
@@ -104,6 +105,10 @@ def embed(blog_id, api_host=None, theme=None):
     # retrieve the wanted theme and add it to blog['theme'] if is not the registered one
     try:
         theme_name = request.args.get('theme', theme)
+        theme = get_resource_service('themes').find_one(req=None, name=blog['blog_preferences'].get('theme'))
```

Comment:
be careful with duplication, this line already exists below.
You should check if it's empty on the one below to prevent another call to the database.